precompiles/gov/gov_test.go

```diff
@@ -96,7 +96,7 @@ func (s *PrecompileTestSuite) TestRun() {
 				Amount:    nil,
 				GasLimit:  100000,
 				GasPrice:  app.MainnetMinGasPrices.BigInt(),
-				GasFeeCap: baseFee,
+				GasFeeCap: baseFee.BigInt(),
```

Comment:
ditto here. This is the fmkt module's base fee. We need to use here the one from the evm module